### ISSUE TITLE
Claim depth reset on transfer.

### ISSUE THREAD

**Issue by NullCase** [RoboMWM/GPAuctions#52]
Test: Create auction, have bidders, let auction end.
Result: Auction completes, claim is transferred. Depth of claim is not retained.
Expected: Retain the claim's depth to secure the property which was secured by the claim being auctioned.

Note: If players auction goods being stored at depth, those goods will not be protected after the claim is transferred.

**Comment by RoboMWM** [comment]
That's very odd. The claim transfer shouldn't modify any of the corner location values so that's strange that isn't being preserved.

Is this being preserved on a /transferclaim ?

**Comment by NullCase** [comment]
Test: create land claim, extend it to depth, transfer the claim.

Result: on /transferclaim depth was preserved.

**Comment by NullCase** [comment]
Perhaps I ought to test the first result again. In case I got it wrong.

It seems like I can do a better job of establishing the claims depth before the auction ends.

Will do tomorrow.

**Comment by NullCase** [comment]
Test 1: Create claim at depth (Y=11), transfer claim. 
Result: Depth retained.

Test 2: Create claim at surface, extend claim to depth (Y=11). Transfer claim. 
Result: Depth retained.

--------------------------

Test 3: Autoauction claim from Test 1. Bid on the auction. Check depth after auction.
Result: Auction claim retains depth (good). After auction claim depth is retained. (wins.)

Test 4: Autoauction claim from Test 2. Bid on the auction. Check depth after auction.
Result: Auction claim retains depth (good). After auction claim depth is retained. (wins.)

# tl;dr

@RoboMWM There is no bug.

It seems like I checked the block _under the Auction sign_. But as mentioned [here](https://github.com/RoboMWM/GPAuctions/issues/50#issuecomment-476003887) signs are being placed in the bottom of a claim. Blocks _beneath_ the bottom of a claim are obviously unclaimed. 

Derp.
